client/python/lib/ovmsclient/tfs_compat/http/serving_client.py

```diff
@@ -103,21 +103,27 @@ def get_model_metadata(self, request):
             >>> response = client.get_model_metadata(request)
             >>> type(response)
         '''
```

Comment:
Can you remove the above docstring? The method behavior is already described in the base class method.